app/models/blog.rb

```diff
@@ -10,8 +10,8 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    terms = { title: "%#{term}%", content: "%#{term}%" }
-    where('title LIKE :title OR content LIKE :content', terms)
+    terms = { term: "%#{term}%" }
+    where('title LIKE :term OR content LIKE :term', terms)
```

Comment:
```suggestion
    where('title LIKE :term OR content LIKE :term', term: "%#{term}%")
```

と1行で書くことが多いですね。

あと、今回はこのままでもいいのですが、`sanitize_sql_like`というメソッドがRailsに用意されているのでそれもちょっと調べてみるといいかもしれません。

Reply:
このままでもいいということでしたが、一行にしました。

[ActiveRecord::Sanitization::ClassMethods](https://api.rubyonrails.org/v7.0/classes/ActiveRecord/Sanitization/ClassMethods.html#method-i-sanitize_sql)

こういうのがあるんですね！`LIKE`以外にもいろいろ用意されてて便利そうって思いました。
余暇で触ってみます〜

Reply:
手元で試してみたらできたので、修正を追加しました〜
https://github.com/ksmxxxxxx/haunted-blog/pull/1/commits/c2920295f2a77fb4d29a074ceeca4f701c95a2e7